lib/Doctrine/ORM/RepositoryException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM;
```

Comment:
I put that at the top level namespace to avoid the repetition. An alternative would be to name the class `Doctrine\ORM\Repository\Exception`, tell me what you prefer.

Reply:
We have a `PersisterException` in `Doctrine\ORM\Persisters` namespace. Also a `CacheException` in `Doctrine\ORM\Cache`. If we keep that structure, it suggests that `RepositoryException` will be in `Doctrine\ORM\Repository`.

I would like to see a structure as you suggested with `Doctrine\ORM\Repository\Exception`, even with the already existing exception classes, but I don't think that this would be an anticipated change.